src/app/(site)/app/post/[postid]/page.tsx

```diff
@@ -32,7 +32,7 @@ export default async function PostPage({
       {postData && !postError ? (
         <div className="flex flex-col gap-1">
           <nav className="w-full pt-4 pb-3 px-5 flex items-center gap-4">
-            <BackwardsNav catchHref="/app" />
```

Comment:
if the user opens a post, shared by a friend, and he clicks the back button, he should be redirected to /app.

the catchHref should be equal to '/app'

Reply:
if the user opens a post from the user page, he would be redirected to /app and no to /search. Whe should fix this

Reply:
by default, the backwards nav redirects you to the last page of your history. the catchHref is a catch, in case the user opens a new tab and it has no history, so instead of redirecting to the google.com page, it redirects to the '/app' route. it was correct as it was before this commit.